app/waf/src/repositories/common.ts

```diff
@@ -15,11 +25,50 @@ export class CommonRepository<
     return super.create(data, options);
   }
 
-  updateById(id: ID, data: DataObject<T>, options?: Options): Promise<void> {
+  find(filter?: Filter<T>, options?: Options): Promise<T[]> {
+    let f: Filter = filter || {};
```

Comment:
I change this line to ```let f = filter || {};```, then I needn't to declare ```f as Filter<T>``` at line 37. Can you please have a try?

Reply:
I guess you test it in control layer

Reply:
No. I tested it repository. Let us refine it in the future.